custom_components/tuya_local/devices/tuya_000004rime_weather_station.yaml

```diff
@@ -0,0 +1,674 @@
+name: Weather station
+products:
+  - id: nghjvn2pcjllwska
+    name: WeatherStation Pro™
+primary_entity:
+  entity: sensor
+  name: Local temperature
+  class: temperature
+  dps:
+    - id: 131
+      type: integer
+      name: sensor
+      unit: C
+      mapping:
+        - scale: 10
+secondary_entities:
+  - entity: sensor
+    name: Weather icon
+    class: enum
+    icon: "mdi:weather-partly-snowy-rainy"
+    dps:
+      - id: 103
+        optional: true
+        type: base64
+        name: sensor
+        mapping:
+          - mask: "000000FF00000000000000000000000000"
```

Comment:
This entry has no value set, which may cause issues with the enum sensor class (which is very fussy about its requirements). A catch-all value of "unknown" would avoid errors in HA, you would just need to check the Payload attribute to see the details of what the unknown value was.